src/python/pants/base/exception_sink.py

```diff
@@ -272,7 +272,7 @@ def exceptions_log_path(cls, for_pid=None, in_dir=None):
     in_dir = in_dir or cls._log_dir
     return os.path.join(
       in_dir,
-      'logs',
+      '.pids',
```

Comment:
One additional level of nesting might be good (`.pids/logs/*`), since there are other directories located in `.pids`.